content/en/blog/2023/adopting-otel-metrics-in-cloud-foundry.md

```diff
@@ -0,0 +1,133 @@
+---
+title: 'Experience Report: Adopting OpenTelemetry for Metrics in Cloud Foundry'
+linkTitle: OpenTelemetry Metrics in Cloud Foundry
```

Comment:
```suggestion
linkTitle: OpenTelemetry at Cloud Foundry
```

Reply:
we shortened this in the latest version, dropping 'Metrics' as you suggested but leaving it as 'in'